examples/snuba-eap-mutations/mutation-1.json

```diff
@@ -1,7 +1,7 @@
 {
   "filter": {
     "organization_id": 1500,
-    "_sort_timestamp": 150,
```

Comment:
Curious why we had to make this change?

Reply:
not strictly necessary, but now there is just one place that computes _sort_timestamp (the snuba consumers), instead of whichever users of the mutations platform we have. `_sort_timestamp` is an attribute derived from `start_timestamp_ms` and IMO should've never been exposed outside of snuba in the first place

Reply:
please check https://github.com/getsentry/snuba/pull/6344/files

Reply:
I think we need to be a little careful here: `_sort_timestamp` exists so to abstract which timestamp values we use to sort the data. So it might not always be start timestamp.

Reply:
@evanh I think we have to either expose _sort_timestamp or start_timestamp_ms to users of the mutation interface. But there's no rush to make this interface change.

Reply:
We should only expose `start_timestamp` to users of the mutation interface and note that it's meant to be a microsecond precision timestamp, not milliseconds.

Reply:
this is how start timestamp is represented in a span on kafka: https://github.com/getsentry/snuba/blob/a612e70b017a95969a6f6f21332952cf5d386b21/rust_snuba/src/processors/eap_spans.rs#L300

intended to match that

what is the interface supposed to be for EAP then?

Reply:
> I think we need to be a little careful here: `_sort_timestamp` exists so to abstract which timestamp values we use to sort the data. So it might not always be start timestamp.

While it's true, I think there's a higher chance we move towards sending incomplete spans from the SDKs and having to update `end_timestamp`, which makes it not viable for the primary key.